src/ethereum/EthereumEngine.ts

```diff
@@ -892,6 +893,66 @@ export class EthereumEngine extends CurrencyEngine<
     }
   }
 
+  getTxParameterInformation(
+    edgeSpendInfo: EdgeSpendInfo,
+    currencyCode: string,
+    currencyInfo: EdgeCurrencyInfo
+  ): EthereumTxParameterInformation {
+    const { spendTargets } = edgeSpendInfo
+    const spendTarget = spendTargets[0]
+    const { publicAddress, nativeAmount } = spendTarget
+
+    if (nativeAmount == null) throw new NoAmountSpecifiedError()
```

Comment:
This check is duplicated

Reply:
It's an assertion to satisfy the type checker.